lib/collection/src/operations/universal_query/planned_query.rs

```diff
@@ -153,9 +161,9 @@ impl PlannedQuery {
                         query,
                         filter,
                         score_threshold,
-                        with_vector: Some(with_vector),
-                        with_payload: Some(with_payload),
-                        offset: 0, // offset is handled at collection level
+                        with_vector: Some(false.into()), // will be fetched after aggregating from segments
+                        with_payload: Some(false.into()), // will be fetched after aggregating from segments
```

Comment:
we can look later into removing these params from `CoreSearchRequest` and `QueryScrollRequest`, but for now we can just disable them here.